src/SeleniumLibrary/__init__.py

```diff
@@ -223,6 +224,89 @@ class SeleniumLibrary(DynamicCore):
 
     See the `Add Location Strategy` keyword for more details.
 
+    = Browser and Window =
+
+    In this section the concept of Browsers or Webdrivers and its Windows
+    will be explained.
+
+    == Browser ==
+
+    In Selenium Library there are two ways to start a Browser and control it.
+    The first one is the `Open Browser` keyword. This one should be used as
+    default keyword.
+    The second one is the `Create Webdriver` keyword which was useful, when
+    browser options needed to be set. Since Selenium Library 4.0.x `Open Browser`
+    also supports option why `Create Webdriver` will be deprecated.
+    When opening a browser a new Webdriver process is startet and connected to
```

Comment:
Would it be good idea to have table, that describer the terms
| WebDriver | Selenium Python class to control browser |
Window, browser, browser driver (example chrome driver) kind a needs their own explanations. 

Also when explaining this thing, would I am starting to think that `browser` is not a good word. Instead it could be cleaner to talk about WebDriver, window and browser driver.

Reply:
i think then it is starting to confuse people again. 

we could explain a lot...
Webdriver is a REST-API and Selenium is the inplementation and a Webdriver Server it the thing at the Browser and the client is Selenium. etc...
Then we could explaint, that Grid nodes are servers and the hub is a Client. 
And nobody would understand whats about windows and Browsers.
i would say, if people want to understand Selenium they should read their docs.

i tried to explain that a "browser" has "windows". I didnt want to explain that a Webdriver instance has a connection to a browserdriver etc...

Reply:
Did refactor this chapter.
More details on Webdriver and Selenium Stuff.